packs/spell-effects/spell-effect-tempest-form.json

```diff
@@ -0,0 +1,88 @@
+{
+    "_id": "mKw9WvqnhaUsPvvy",
+    "img": "systems/pf2e/icons/spells/tempest-form.webp",
+    "name": "Spell Effect: Tempest Form",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Tempest Form]</p>\n<p>Your body becomes fluid to better suit your surroundings. When you Cast this Spell, choose whether to become air, water, or mist. The spell gains the air trait if you choose air or mist, and the water trait if you choose water or mist. You become amorphous, as does your armor. You lose any item bonus to AC and use your proficiency bonus for unarmored defense to determine your AC. You also gain resistance 10 to physical damage and become immune to precision damage. You can slip through tiny cracks and don't need to breathe. You can't cast spells, activate items, or use actions that have the attack or manipulate trait, except those granted by this spell. You also gain the following effects based on your form.</p>\n<ul>\n<li><strong>Air</strong> You gain a fly Speed of 20 feet and become invisible while you are in the air. You can create the effects of a <em>gust of wind</em> from your space as a 2-action activity, which has the manipulate trait.</li>\n<li><strong>Mist</strong> You gain a fly Speed of 20 feet, and it becomes hard to see through you. Any creature on one side of your space who is targeted by a creature on the opposite side is concealed to the targeting creature.</li>\n<li><strong>Water</strong> You gain a swim Speed of 20 feet and become invisible while you are in the water. You can electrically charge yourself by taking a single action, which has the manipulate trait. If you do, you are no longer invisible in the water due to electricity indicating your location, but any creature that makes a melee attack against you takes 1d6 electricity damage; this is cumulative with the damage from your major curse in situations where both apply.</li>\n</ul>\n<hr />\n<p><strong>Heightened (+2)</strong> Increase the resistance by 5 and the electricity damage from the charged water form by 1.</p>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": true,
+            "unit": "minutes",
+            "value": 1
+        },
+        "level": {
+            "value": 6
+        },
+        "publication": {
+            "license": "OGL",
+            "remaster": false,
+            "title": "Pathfinder Advanced Player's Guide"
+        },
+        "rules": [
+            {
+                "choices": [
+                    {
+                        "label": "PF2E.SpecificRule.TempestForm.Air",
```

Comment:
We could use PF2E.TraitAir and PF2E.TraitWater, but that might cause translation issues in some locales.